rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -2269,6 +2304,25 @@ error_code sys_net_bnet_recvfrom(ppu_thread& ppu, s32 s, vm::ptr<void> buf, u32
 
 				return true;
 			}
+#ifdef _WIN32
+			else
+			{
+				// Windows returns an error when trying to peek at a message and buffer not long enough to contain the whole message, should be ignored
+				if ((native_flags & MSG_PEEK) && get_native_error() == WSAEMSGSIZE)
+				{
+					native_result = len;
+					WSASetLastError(0);
```

Comment:
Those 2 WSASetLastError(0); are not needed, we only check error on api call failure and by then the error would have been set to the new one.
Apart from that I'm fine with this PR.

Reply:
Will we hold a separate per-thread error value that we explicitly update in each net function to allow us to handle the _sys_net_errno_loc() call eventually?
The WSASetLastError(0) was just 'stolen' from cygwin, I figured we'd be implementing pretty much the same kind of thing as they have (translation from posix <-> native Win32), but that was on the assumption there would be a PS3 get_errno type of function call (rather than a function to return a pointer to the errno value)

Reply:
I've not delved too deep into this but sys_net_errno is not a syscall, it's part of a firmware library. It's probably updated depending on return values from the syscalls and handled by the library.

Reply:
Ok, I've dropped them out for now then.

Do we have any standard way of tracking the address offsets into TLS?
I see that for the _sys_net_errno we have "ppu.gpr[13] - 0x7030 + 0x2c", and I've seen similar around.
My perhaps naïve thought was that an enum or similar for the TLS storage variables (offset) would have allowed better traceability, or is it really not known at compile time?

Reply:
_sys_net_errno is part of libnet which is lled, which is why most functions are unimplemented, like I said previously it doesn't matter.